openemory/publication/templates/publication/upload.html

```diff
@@ -156,7 +156,19 @@ <h2>Mediated Deposit</h2>
            {% endif %}
         </div>
 
-        <div>
+        <div class="margin-15">
+          <label for="{{ form.content_model.field.name }}" class="tip">{{ form.content_model.label }}
+          
+          {% if form.content_model.help_text %}
+            <a class="tip" title="{{ form.content_model.help_text }}"></a>
+            {% endif %}
+            {{ form.content_model }}
+           {% if form.content_model.errors %}
+             <p class="error">{{ form.content_model.errors }}</p>
+           {% endif %}
+        </div>
```

Comment:
Not sure I understand this part, but nothing looks out of the ordinary.

Reply:
This is how the dropdown for content_model is displayed. Help text goes into a tooltip for people to get directions.

Reply:
Oh! I see how this works now. Thanks!